tests/std/tests/P0645R10_text_formatting_formatting/test.cpp

```diff
@@ -340,8 +349,398 @@ void test_multiple_replacement_fields() {
+    assert(format(STR("{:#a} {:#a}"), inf, nan) == STR("inf nan"));
+    assert(format(STR("{:#A} {:#A}"), inf, nan) == STR("INF NAN"));
+    assert(format(STR("{:#e} {:#e}"), inf, nan) == STR("inf nan"));
+    assert(format(STR("{:#E} {:#E}"), inf, nan) == STR("INF NAN"));
+    assert(format(STR("{:#f} {:#f}"), inf, nan) == STR("inf nan"));
+    assert(format(STR("{:#F} {:#F}"), inf, nan) == STR("inf nan"));
+    assert(format(STR("{:#g} {:#g}"), inf, nan) == STR("inf nan"));
+    assert(format(STR("{:#G} {:#G}"), inf, nan) == STR("INF NAN"));
+
+    // Width
+    assert(format(STR("{:3}"), Float{0}) == STR("  0"));
+    assert(format(STR("{:#9G}"), Float{12.2}) == STR("  12.2000"));
+    assert(format(STR("{:#12g}"), Float{1'000'000}) == STR(" 1.00000e+06"));
+
+    // Precision
+    Float value = 1234.52734375;
+    assert(format(STR("{:.4}"), value) == STR("1235"));
+    assert(format(STR("{:.1}"), inf) == STR("inf"));
+    assert(format(STR("{:.1}"), nan) == STR("nan"));
+
+    assert(format(STR("{:.4a}"), value) == STR("1.34a2p+10"));
+    assert(format(STR("{:.4A}"), value) == STR("1.34A2P+10"));
+
+    assert(format(STR("{:.4e}"), value) == STR("1.2345e+03"));
+    assert(format(STR("{:.4E}"), value) == STR("1.2345E+03"));
+
+    assert(format(STR("{:.4f}"), value) == STR("1234.5273"));
+    assert(format(STR("{:.4F}"), value) == STR("1234.5273"));
+
+    assert(format(STR("{:.4g}"), value) == STR("1235"));
+    assert(format(STR("{:.4G}"), value) == STR("1235"));
+
+    // Leading zero
+    assert(format(STR("{:06}"), Float{0}) == STR("000000"));
+    assert(format(STR("{:06}"), Float{1.2}) == STR("0001.2"));
+    assert(format(STR("{:06}"), nan) == STR("   nan"));
+    assert(format(STR("{:06}"), inf) == STR("   inf"));
+
+    // Type
+    assert(format(STR("{:a}"), value) == STR("1.34a1cp+10"));
+    assert(format(STR("{:A}"), value) == STR("1.34A1CP+10"));
+
+    assert(format(STR("{:e}"), value) == STR("1.234527e+03"));
+    assert(format(STR("{:E}"), value) == STR("1.234527E+03"));
+
+    assert(format(STR("{:f}"), value) == STR("1234.527344"));
+    assert(format(STR("{:F}"), value) == STR("1234.527344"));
+
+    assert(format(STR("{:g}"), value) == STR("1234.53"));
+    assert(format(STR("{:G}"), value) == STR("1234.53"));
+}
+
+template <class charT>
+void test_pointer_specs() {
+    assert(format(STR("{:}"), nullptr) == STR("0x0"));
+
+    // Sign
+    throw_helper(STR("{: }"), nullptr);
+    throw_helper(STR("{:+}"), nullptr);
+    throw_helper(STR("{:-}"), nullptr);
+
+    // Alternate form
+    throw_helper(STR("{:#}"), nullptr);
+
+    // Leading zero
+    throw_helper(STR("{:0}"), nullptr);
+
+    // Width
+    assert(format(STR("{:5}"), nullptr) == STR("0x0  "));
+
+    // Precision
+    throw_helper(STR("{:.5}"), nullptr);
+
+    // Types
+    assert(format(STR("{:p}"), nullptr) == STR("0x0"));
+    throw_helper(STR("{:a}"), nullptr);
+}
+
+template <class charT>
+void test_string_specs() {
+    auto cstr = STR("foo");

```

Comment:
We conventionally avoid "foo" in test code. (It's not forbidden by policy, but avoiding it makes following policy easier.)

If you have a cat/dog/fish/gerbil/etc. here's a perfect opportunity for them to be the star:
https://github.com/microsoft/STL/blob/c5ee3b32bd3ee219b6d93d72f24429ae6e240ced/tests/std/tests/Dev11_0000000_user_defined_literals/test.cpp#L295

Reply:
I can change it, but I am not sure what you mean. If it's not forbidden by policy, which policy does it make following easier?